app/src/main/java/com/example/plannr/admin/Course.java

```diff
@@ -0,0 +1,44 @@
+package com.example.plannr.admin;
```

Comment:
this file concerns both admins and students, so it should not belong in the admin folder. i would suggest a `course` folder where we put all files related to courses